gph/python/ripser_interface.py

```diff
@@ -444,6 +459,13 @@ def ripser_parallel(X, maxdim=1, thresh=np.inf, coeff=2, metric="euclidean",
             "`collapse_edges` and `return_generators`cannot both be True."
         )
 
+    if coeff != 2 and \
+            not _is_prime_and_larger_than_2(
+                coeff, gph_ripser.get_max_coefficient_field_supported()):
```

Comment:
Maybe store `gph_ripser.get_max_coefficient_field_supported()` into a variable so we don't call it twice?

Reply:
The computation time is almost null.
But it will definitely make the line smaller, let me do the change :+1:

Reply:
Done, I let you resolve the conversation if it is good for you

Reply:
Thanks!